src/ext/ca/wixca/dll/qtexecca.cpp

```diff
@@ -62,17 +94,17 @@ HRESULT BuildCommandLine(
 
 #define ONEMINUTE 60000
 
-DWORD GetTimeout()
+DWORD GetTimeout(LPCSTR szPropertyName, LPCWSTR wzPropertyName)
 {
     DWORD dwTimeout = ONEMINUTE;
     HRESULT hr = S_OK;
 
     LPWSTR pwzData = NULL;
 
-    if (WcaIsPropertySet("QtExecCmdTimeout"))
+    if (WcaIsPropertySet(szPropertyName))
```

Comment:
Will WcaIsUnicodePropertySet work here instead? I :dizzy_face: at the duplication.